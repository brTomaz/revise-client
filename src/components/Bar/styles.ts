import styled, { css } from 'styled-components'

export const Wrapper = styled.aside`
  ${({ theme }) => css`
    width: 100%;
    height: 6.4rem;
    background: linear-gradient(
      120deg,
      ${theme.colors.secondary} 0%,
      ${theme.colors.primary} 100%
    );
  `}
`
