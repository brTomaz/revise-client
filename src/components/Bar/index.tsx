import * as Styles from './styles'

const Bar = () => <Styles.Wrapper />

export default Bar
