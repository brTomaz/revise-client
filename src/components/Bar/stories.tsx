import { Story, Meta } from '@storybook/react/types-6-0'
import Bar from '.'

export default {
  title: 'Bar',
  component: Bar
} as Meta

export const Default: Story = () => <Bar />
