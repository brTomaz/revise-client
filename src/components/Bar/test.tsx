import { renderWithTheme } from 'utils/tests/helpers'
import Bar from '.'

describe('<Bar />', () => {
  it('should render Bar with styles', () => {
    const { container } = renderWithTheme(<Bar />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
