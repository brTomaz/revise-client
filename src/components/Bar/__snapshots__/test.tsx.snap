// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Bar /> should render Bar with styles 1`] = `
.c0 {
  width: 100%;
  height: 6.4rem;
  background: linear-gradient( 120deg,#2DC2C3 0%,#2D3D88 100% );
}

<aside
  class="c0"
/>
`
